src/PowerShellEditorServices/Services/TextDocument/Handlers/ReferencesHandler.cs

```diff
@@ -22,7 +22,7 @@ class ReferencesHandler : IReferencesHandler
         private readonly ILogger _logger;
         private readonly SymbolsService _symbolsService;
         private readonly WorkspaceService _workspaceService;
-        private ReferencesCapability _capability;
+        private ReferenceCapability _capability;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this unread private field '_capability' or refactor the code to use its value.](https://app.codacy.com/manual/TylerLeonhardt/PowerShellEditorServices/pullRequest?prid=5191664)